sdcm/stress_thread.py

```diff
@@ -110,10 +112,15 @@ def create_stress_cmd(self, node, keyspace_idx):
         elif 'keyspace=' not in stress_cmd:  # if keyspace is defined in the command respect that
             stress_cmd = stress_cmd.replace(" -schema ", " -schema keyspace=keyspace{} ".format(keyspace_idx))
 
+        if self.sla_role and '<role ' in stress_cmd:
+            # put the credentials with SLA role, replace <role n>
+            stress_cmd = re.sub(r'<role \d>', f'user={self.sla_role} password={self.sla_role}', stress_cmd)
```

Comment:
role name is also it's password ? it's a bit confusing.

Also this regex won't catch <role 10>

Why don't we do this replacement first time we scan the stress command ? why we need todo it in two places ?

Reply:
changed